mast_aladin_lite/app.py

```diff
@@ -1,20 +1,62 @@
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from astropy.utils.masked import Masked
+
 from ipyaladin import Aladin
 from mast_aladin_lite.aida import AID
+from mast_aladin_lite.table import MastTable
 
 
 # store reference to the latest instantiation:
 _latest_instantiated_app = None
 
 
 class MastAladin(Aladin):
+    selected_table = None
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+
+        # the `aid` attribute gives access to methods from the
+        # Astro Image Display (AID) API
         self.aid = AID(self)
 
         global _latest_instantiated_app
         _latest_instantiated_app = self
 
+    def load_table(self, table, name='table', load_footprints=True, update_viewport=True):
+        table_widget = MastTable(table)
+        self.selected_table = table_widget
+
+        if load_footprints:
+            if 's_region' in table.colnames:
+                self.add_graphic_overlay_from_stcs(table['s_region'])
+            else:
+                raise ValueError(
+                    "The table does not contain an `s_region` column, so no "
+                    "footprints can be loaded."
+                )
+
+        if update_viewport:
+            ra_column, dec_column = 'ra', 'dec'
+
+            if ra_column not in table.colnames:
+                ra_column, dec_column = 'targ_ra', 'targ_dec'
+
+            mask = (table[ra_column] == 0) & (table[dec_column] == 0)
+
+            center_coord = SkyCoord(
```

Comment:
I suggest using the RA and Dec of the first unmasked row instead of the average.  In the case where the MAST query was not a cone search or searched a list of targets, the average may put Aladin's target not near any of the results.  

If the MAST query was a simple cone search, then MAST's target will still include results even if not ideally centered.  In that case, the user could specify an optional `default_display_position` in the `MastTable` constructor using the same target they used in their search.